### DISCUSSION TITLE
Order of next steps to publish the 'trilogy'

### DISCUSSION THREAD

**Discussion by j-hagedorn** [j-hagedorn/trilogy#47] [Q&A]
Hey @sdaranyi  and @salmonix , I put the issues in [our next milestone](https://github.com/j-hagedorn/trilogy/milestone/4) in an order which I think makes sense.  Please let me know your thoughts, when you get the chance.  Here's the basic idea:

1. Clean up the existing flattened files for the TMI and ATU
2. Merge the AT with ATU
3. Make the graph datasets based on the cleaned versions
4. Document it all
5. Clean up the AFT if possible

If any of these is unnecessary, please note it in the issue, and I'll gladly remove the issue from this milestone to streamline our work prior to the conference.

**Comment by sdaranyi** [comment]
Does this mean that you managed to merge the AT with the ATU, or we ignore
that for now? The former would be quite a development.

Sándor

**Reply by j-hagedorn** [reply]
No, we deprioritized that a few months back after my initial communication here.

**Comment by salmonix** [comment]
Which text shall be used for manual re-tagging?
L.

**Reply by j-hagedorn** [reply]
@salmonix , do you mean for the ATU?  The tale type descriptions are in 'atu_df'

**Comment by salmonix** [comment]
Not exactly. I want to add a manual resolution for ambiguous texts. See: branch notes_on_atu -> docs/ATU_NOTES.md
I would like to resolve the ambiguities manually.
Side not: what is the character encoding for the text files? On my Linux settings many accented characters are scrambled. Eg. data/atu_df.csv: "Hungarian: Gyï¿½rgy"  instead of "Hungarian: György"

**Reply by j-hagedorn** [reply]
I imagine you'd want the source text file referenced here: https://github.com/j-hagedorn/trilogy/blob/master/fetch/fetch_taletypes.R#L5

@sdaranyi may be able to provide it, or I can send once I'm at my computer.

Would love if you can identify the encoding issue so that I can fix those issues with characters!

**Comment by sdaranyi** [comment]
Sure if necessary.
